src/mumble/GlobalShortcut.ui

```diff
@@ -216,6 +216,16 @@
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
+          <widget class="QCheckBox" name="qcbEnableUIAccess">
+           <property name="whatsThis">
+            <string>&lt;b&gt;Enable Windows UI automation&lt;/b&gt;.&lt;br /&gt;Also known as &quot;UIAccess&quot;. This allows Mumble to recieve global shortcut events from programs running at high privilege levels.</string>
```

Comment:
Typo recieve. Also never heard it called Windows UI automation. Isn't it Windows UI accessibility?

Imho it would be helpful if this would also mention why you might want to use this option.